fabfile.py

```diff
@@ -13,13 +13,41 @@
 CURRENT_DIR = os.path.dirname(__file__)
 
 
+TIERS = {
+    'production': {
+        'hosts': ['root@104.131.92.59'],
```

Comment:
This is weird to me, but I guess ip addresses are not really secret. Though I do still feel pretty odd about the fact that we include the username here.